erpnext/utilities/user_progress.py

```diff
@@ -81,6 +81,26 @@ def get_slide_settings():
 				}
 			]
 		),
+
+		frappe._dict(
+			action_name='Add Letterhead',
+			domains=('Manufacturing', 'Services', 'Retail', 'Distribution', 'Education'),
+			title=_("Add Letterhead"),
+			help=_("Upload your letter head. (you can edit it later)."),
+			fields=[
+				{"fieldtype":"Attach Image", "fieldname":"letterhead",
+					"label":_("Letterhead"),
+					"description": _("Keep it web friendly 900px (w) by 100px (h)"),
```

Comment:
Let's just skip the label and description altogether. Will also help in keeping the image control in the centre. Instead, we'll change help to "Upload your letter head (Keep it web friendly as 900px by 100px)". Once inside, they'll already be aware that things can be changed later on. (The edit line was while in the setup wizard).